vnet/lib/vnet/endpoints/1.0/datapaths.rb

```diff
@@ -41,16 +41,21 @@ def self.put_post_shared_params
   end
 
   param_uuid M::Interface, :interface_uuid, required: true
-  param :broadcast_mac_address, :String, required: true, transform: PARSE_MAC
+
+  # 'broadcast_mac_address' is deprecated and should be phased out.
+  param :mac_address, :String, transform: PARSE_MAC
```

Comment:
`broadcast_mac_address` used to be required but `mac_address` isn't. What happens when a mac address isn't provided?